corehq/apps/app_manager/views/releases.py

```diff
@@ -423,7 +423,8 @@ def revert_to_copy(request, domain, app_id):
                 action=ApplicationReleaseLog.ACTION_REVERTED,
                 version=app.version,
                 app_id=app_id,
-                user_id=user_id
+                user_id=user_id,
+                info=_(f"Reverted to version {copy_build_comment_params['old_version']}")
```

Comment:
I don't think we can translate this here because then it would keep it in language of the user that reverted it, and not for the user that views it.
I think we should keep info as a dict and then it can be rendered based on the action.